assets/css/style.scss

```diff
@@ -1,4 +1,6 @@
 ---
 ---
 
-@import "plain";
\ No newline at end of file
+@import "plain";
+@import "toggle";
+@import "dark";
```

Comment:
We could wrap this in a if block with `site.plainwhite.dark_mode` so that we load only the css which is required.